specification/elastic/resource-manager/Microsoft.Elastic/preview/2023-02-01-preview/elastic.json

```diff
@@ -1371,7 +1417,7 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Elastic/getOrganizationApiKey": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Elastic/getOrganizationApiKey": {
```

Comment:
>getOrganizationApiKey [](http://example.com/codeflow?start=65&length=21)

if u call this listOrganizationApiKey, you can let your customers use this in ARM templates

Reply:
@rkmanda, did you mean to change the API name from "getOrganizationApiKey" to "listOrganizationApiKey" ?
Apologies, the comment is a tad confusing.

Reply:
If yes, updated and renamed API as suggested in https://github.com/Azure/azure-rest-api-specs/pull/23343/commits/bfd1a05f2352a3b650b258f1ae2428a531036318